### PR TITLE
[retiarii] use fixed seeds for unit test of CGO engine

### PR DESCRIPTION
**Pull request by hzhua**
The validation accuracy for the unit test may be lower than 0.8 with some initialized weights. Use fixed seeds instead.

### CODE REVIEW

**Review by ultmaster — APPROVED** [review]
In Pytroch-lightning they have an API called `seed_everything`

File: test/ut/retiarii/test_cgo_engine.py [hunk 1]
```diff
@@ -5,6 +5,9 @@
 import time
 import torch
 import torch.nn as nn
+import numpy as np
+import random
```
Comment (liuzhe-lz) [inline test/ut/retiarii/test_cgo_engine.py:9]: Is this required?

Comment (hzhua) [reply test/ut/retiarii/test_cgo_engine.py]: removed.

**Review by liuzhe-lz — APPROVED** [review]
